chrome/content/brief-overlay.js

```diff
@@ -192,12 +194,14 @@ var Brief = {
             Brief.constructTooltip();
     },
 
-    updateStatus: function Brief_updateStatus() {
-        if (!Brief.toolbarbutton)
+    updateStatus: function Brief_updateStatus(toolbarbutton) {
+        if (toolbarbutton == undefined)
+            toolbarbutton = Brief.toolbarbutton.forWindow(window);
```

Comment:
This looks wrong taking into account that updateStatus is frequently called from a window-independent context (e.g. onEntriedAdded handler).